akvo/rsr/static/scripts-src/my-results/components/updates/UpdateForm.jsx

```diff
@@ -934,12 +959,27 @@ export default class UpdateForm extends React.Component {
     updateActionsHandler(e) {
         //The id of the button is used to indicate the action taken
         const action = e.target.id;
-        if (action === c.UPDATE_ACTION_DELETE) {
-            this.deleteUpdate();
-        } else {
-            this.saveUpdate(action);
-        }
+        switch (action) {
+            case c.UPDATE_ACTION_DELETE: {
+                this.deleteUpdate();
+                break;
+            }
+
+            case c.UPDATE_ACTION_EDIT: {
```

Comment:
I think this case and the next never happen.

Reply:
Yes, I had reverted a commit, and seems like git revert didn't revert this bit.